.github/workflows/build-image.yml

```diff
@@ -52,4 +57,4 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
-          platforms: linux/amd64,linux/arm64
+          platforms: ${{ matrix.platform }}
```

Comment:
The only benefit is concurrency. Multi-platform images can be built and pushed simultaneously.

Reply:
@aktech sorry for the late reply. The reason I asked this is because [on the docker images repo we're using the multi-platform strategy instead of using a matrix](https://github.com/nebari-dev/nebari-docker-images/blob/e75a1650d07b8b138301c9243fe4062fa5a40004/.github/workflows/build-push-docker.yaml#L122). If the end result is the same, I don't mind which one we're using.

Reply:
Yep same result, I didn't have the chance to change on the the main (docker images) repo.